plugins/woocommerce/tests/e2e-pw/tests/shopper/launch-your-store.spec.js

```diff
@@ -72,43 +72,51 @@ async function runComingSoonTests( themeContext = '' ) {
 	} );
 }
 
-test.describe( 'Launch Your Store front end - logged out', () => {
-	test.afterAll( async ( { baseURL } ) => {
-		try {
-			await setOption(
-				request,
-				baseURL,
-				'woocommerce_coming_soon',
-				'no'
-			);
-		} catch ( error ) {
-			console.log( error );
-		}
-	} );
-
-	test.describe( 'Block Theme (Twenty Twenty Four)', () => {
-		test.beforeAll( async () => {
-			await activateTheme( 'twentytwentyfour' );
+test.describe(
+	'Launch Your Store front end - logged out',
+	{ tag: [ '@skip-on-default-wpcom', '@skip-on-default-pressable' ] },
```

Comment:
The LYS feature is not present on WPCOM (they are working on it as a project), and for Pressable, we are skipping until we figure out how to execute CLI commands. Currently, we have the wp-env CLI, which works only on wp-env. This is to change the active theme in the test.

Reply:
Oh, wow.. I didn't know that existed. We may consider it at least for Pressable, but I hope we can find a way for all envs, might be good for a separate PR.